package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Javascript API to talk to Nextcloud",
   "main": "compiled/client.js",
   "scripts": {
-    "build": "npx esbuild --bundle --platform=node --target=es2018 source/client.ts --outdir=compiled --minify",
+    "build": "npx esbuild --bundle --platform=node --target=es2018 source/client.ts --outdir=compiled --minify && npx typescript source/*.ts source/**/*.ts --declaration --emitDeclarationOnly --outDir compiled/source/",
```

Comment:
What is that second part? Generating type declarations?
What about creating a separate task and then changing or adding another task that does `npm run x && npm run types:generate` ?